### PR TITLE
Added functional ORU spider

### PR DESCRIPTION
**Pull request by HarrisonGregg**

### CODE REVIEW

File: osp_scraper/spiders/oru.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                        meta={
+                            'source_anchor': name
+                        },
+                        callback=get_departments
+                    )
+                    
+        def get_departments(response):
+            for option in response.css("select[name='dept'] option"):
+                code = option.css('option::attr(value)').extract_first()
+                if code:
```
Comment (jodizzle) [inline osp_scraper/spiders/oru.py]: Are these `if code` conditionals used to check against the possibility of the empty-value options at the beginning of the lists like "Select a Term"?  If so, I would just slice those entries off the list and save yourself some indentation, e.g.,
```python
for option in response.css("select[name='dept'] option")[1:]:
```

File: osp_scraper/spiders/oru.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                            'depth': 1,
+                            'hops_from_seed': 1
+                        },
+                        callback=self.parse_for_files
+                    )
+
+        yield scrapy.Request(get_syllabi_url, callback=get_terms)
+
+    def extract_links(self, response):
+        for a in response.css('.main-content form a'):
```
Comment (jodizzle) [inline osp_scraper/spiders/oru.py]: Minor: Instead of `a`, I would use `tag` as the variable name like you have in some other scrapers.

Comment (HarrisonGregg) [inline osp_scraper/spiders/oru.py]: Done.

File: osp_scraper/spiders/oru.py (hunk 1 above)
Comment (HarrisonGregg) [inline osp_scraper/spiders/oru.py]: I used to slice them off that way, but I settled on this pattern to be as robust as possible.  It definitely isn't _necessary_, but will handle some unexpected cases more cleanly.

**Pull request merged**